app/controllers/sessions_controller.rb

```diff
@@ -30,6 +31,10 @@ def callback
   end
 
   def destroy
+    session[ActionInterceptor.config.default_key] = nil
+    session[:registration_return_to] = nil
+    session[:client_id] = nil
```

Comment:
You can't guarantee that the session vars will be cleared, because you can't guarantee that the user will ever come here.